env.sh

```diff
@@ -45,6 +45,20 @@ action() {
         run_cmd scramv1 project CMSSW $CMSSW_VER
         run_cmd cd $CMSSW_VER/src
         run_cmd eval `scramv1 runtime -sh`
+        run_cmd git-cms-init
+        run_cmd git clone ssh://git@gitlab.cern.ch:7999/akhukhun/roccor.git RoccoR
+        run_cmd git clone git@github.com:SVfit/ClassicSVfit.git TauAnalysis/ClassicSVfit -b fastMTT_19_02_2019
+        run_cmd git clone git@github.com:SVfit/SVfitTF.git TauAnalysis/SVfitTF
+        run_cmd git clone ssh://git@gitlab.cern.ch:7999/rgerosa/particlenetstudiesrun2.git ParticleNetStudiesRun2 -b cmssw_124X
```

Comment:
this is repository for training - we need only model files => after files are copied this repository can be deleted

Reply:
I could see the first three not being necessary (I think those aren't even copied to CMSSW in a crab job), but I'd want to check it myself first, just to be sure. I was just following the [installation recipe](https://gitlab.cern.ch/rgerosa/particlenetstudiesrun2/-/blob/master/README.md) from ParticleNet.
The last one I'm pretty sure is necessary, because it contains the EDProducer `ParticleNetFeatureEvaluator`